daily_jobs.py

```diff
@@ -132,30 +133,28 @@ def add_zoom_registrants_to_mailgun():
     meetings = common.all_meetings("me")
 
     # Filter by past meetings
-    past_lrc = [
+    time_now = datetime.datetime.now(tz=pytz.UTC)
+    time_yesterday = time_now - datetime.timedelta(days=60)
```

Comment:
Turtle days :thinking: :laughing: 

```suggestion
    time_yesterday = time_now - datetime.timedelta(days=1)
```

Reply:
Turtle? :P I fixed that right when you were posting this probably 😬